### ISSUE TITLE
Add proxy priority changes using dragging

### ISSUE THREAD

**Issue by pointydev** [younesaassila/ttv-lol-pro#92]
Hey there,

Would be nice if sorting the list of proxy priority could be done by dragging them instead of having to manually add and remove entries in reverse order.

Thanks,
pointy

**Comment by thinkpad4** [comment]
YES, this.  That would be very helpful.

**Comment by juliocoliveira** [comment]
Yes it would be great, By Dragging or Arrows (Up/Down) to change the order.
I made a similar suggestion #115 because didn't know that this as created too.
